components/script/dom/webglrenderingcontext.rs

```diff
@@ -77,7 +79,7 @@ use pixels::{self, PixelFormat};
 use script_layout_interface::HTMLCanvasDataSource;
 use serde::{Deserialize, Serialize};
 use servo_config::pref;
-use std::cell::Cell;
+use std::cell::{Cell, Ref};
```

Comment:
I suspect we might need to import dom::bindings::cell::Ref instead?

Reply:
Yes, you are right! Thank you!